score-client/src/main/resources/application.yml

```diff
@@ -36,17 +36,6 @@ client:
   ssl:
     custom: false
 
-kfportal:
-  url: https://arranger.kids-first.io/june_13/graphql
-
-kf:
-  token:
-    url: https://v2i1r42t6d.execute-api.us-east-1.amazonaws.com/beta/token
-  download:
-    url: https://data.kidsfirstdrc.org
-    partSize: 1073741824
```

Comment:
Test fails. `StorageConfig` cannot  resolve find the value of `kf.download.partSize`